pkgs/misc/tmux-plugins/default.nix

```diff
@@ -127,6 +127,17 @@ in rec {
     };
   };
 
+  dracula = mkDerivation rec {
+    pluginName = "dracula";
+    version = "1.0.1";
+    src = fetchFromGitHub {
+      owner = "dracula";
+      repo = "tmux";
+      rev = "cb1d420a6267c600a8ef0e7a3c00e2474b30ae99";
+      sha256 = "1hz2grgq6h9lb0n33rl12qjslnky6zs4r494fcb9h08vym9z9sj0";
+    };
```

Comment:
:books: Would it be possible to add the meta attributes, you can look at [resurrect](https://github.com/NixOS/nixpkgs/blob/440c5808a18980909ca2eb349d4819d266967e49/pkgs/misc/tmux-plugins/default.nix#L294-L319) as an example (omit `longDescription` if unavailable).

You will also need to add yourself as a maintainer of this package. Add yourself to the maintainer list which you can find [here](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix). It should be updated in its own commit with the message `maintainers: add <username>`.

You can find your `githubId` with `curl -s https://api.github.com/users/kyleondy | jq '.id'`

Reply:
Done 👍